mp/game/momentum/resource/momentum_english_ref.res

```diff
@@ -90,6 +90,7 @@
     //Timer
         "MOM_SavedLocation" "Saveloc %s1/%s2"
         "MOM_Cheater_Message" "For your times to be able to submit,\nyou must turn sv_cheats off!"
+        "MOM_Tickrate_Warning" "For your times to be able to submit,\nyou must use sv_tickrate 66!"
```

Comment:
Because not all game modes default to 66 tick, we will want to generify this:
```suggestion
        "MOM_Tickrate_Warning" "For your times to be able to submit,\nyou must use the default tickrate for this gamemode!"
```